openshift/certbot/docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# cat Dockerfile | oc new-build -D - --name=certbot
```

Comment:
Minor: commented out command

Reply:
This is taken directly from the certbot project - so I'd rather not change it if I don't need to.